CMakeLists.txt

```diff
@@ -31,6 +31,15 @@ if (VUL_WERROR)
     endif()
 endif()
 
+option(VVL_CLANG_TIDY "Use clang-tidy")
+if (VUL_CLANG_TIDY)
+    find_program(CLANG_TIDY NAMES clang-tidy)
+    if(NOT CLANG_TIDY)
+        message(FATAL_ERROR "clang-tidy not found!")
+    endif()
+    set(CMAKE_CXX_CLANG_TIDY "${CLANG_TIDY}")
```

Comment:
CMAKE_\<LANG\>_CLANG_TIDY only works on the makefile / ninja generator.

Given how popular the visual studio generator is within our team this is likely only going to cause confusion.